lib/index.js

```diff
@@ -8,11 +8,13 @@ var Options = require('./options'),
     bundle = require('./bundle'),
     dereference = require('./dereference'),
     url = require('./util/url'),
+    getMeta = require('./util/meta').getMeta,
     maybe = require('call-me-maybe'),
     ono = require('ono');
 
 module.exports = $RefParser;
 module.exports.YAML = require('./util/yaml');
+module.exports.getMeta = getMeta;
```

Comment:
Let's rename `getMeta()` to `getMetadata()`.  It's more explicit/clear.

```suggestion
module.exports.getMetadata = getMetadata;
```